extra-webpack.config.js

```diff
@@ -23,7 +23,7 @@ module.exports = {
   ],
   resolve: {
     fallback: {
-      bootstrap: path.join(__dirname, 'projects/styles/bootstrap-copy'),
+      bootstrap: path.join(__dirname, 'projects/storefrontstyles/bootstrap-copy'),
```

Comment:
`minor:` this `bootstrap` fallback alias is no longer needed as there is no direct reference to `'bootstrap'` location in our app anymore. Bootstrap copy is part of `@spartacus/styles` (handled by the config below).